go/saltpack/const.go

```diff
@@ -39,6 +39,22 @@ const EncryptionArmorHeader = "BEGIN KEYBASE ENCRYPTED MESSAGE"
 // armored KB message.
 const EncryptionArmorFooter = "END KEYBASE ENCRYPTED MESSAGE"
 
+// SignedArmorHeader is the header that marks the start of signed
+// armored KB message.
+const SignedArmorHeader = "BEGIN KEYBASE SIGNED MESSAGE"
```

Comment:
May as well change it over to the right text:

`BEGIN KEYBASE SALTPACK SIGNED MESSAGE`